### ISSUE TITLE
How to disable "Restart FF after OS restart" banner on first start

### ISSUE THREAD

**Issue by htcfreek** [mozilla/policy-templates#1146]
Ho can I disable the "Open Firefox after each OS restart. Now you can configure Firefox to open automatically on restarting your device." banner that appears after updating to the latest Firefox ESR (128.*) and on first startup of Fifrefox?

![image](https://github.com/user-attachments/assets/1aba2bdf-6ef3-4ae6-8cdc-2cf8ea2b1351)

**Comment by mkaply** [comment]
browser.startup.windowsLaunchOnLogin.disableLaunchOnLoginPrompt
for the prompt
browser.startup.windowsLaunchOnLogin.enabled
for the feature.

You can use the preference policy to set it:

https://mozilla.github.io/policy-templates/#preferences

**Comment by htcfreek** [comment]
@mkaply 
Only for clarification ...

> browser.startup.windowsLaunchOnLogin.disableLaunchOnLoginPrompt for the prompt 

Setting this to `true` prevents the banner from opening.

> browser.startup.windowsLaunchOnLogin.enabled for the feature.

Setting this to `true` does nothing. Setting this to `false` tells FF that this feature is not available/supported and hides it from the ui. And this entry does not affect the setting in preferences so I can not enable this feature by this preference?

**Comment by mkaply** [comment]
Do you want to enable the feature?

**Comment by mkaply** [comment]
Yeah, there is no pref to control the feature. It's done in the registry from what I can tell.
